src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/KalbachMann/Data/Builder.hpp

```diff
@@ -0,0 +1,32 @@
+template< typename ParentBuilder >
+class Builder {
+
+  std::reference_wrapper< ParentBuilder > parent;
+
+  std::optional< int > interpolationParameter_;
```

Comment:
So this one can also have a number of discrete photons. As with LAW=4, maybe introduce the numberDiscretePhotons and limit INTT to 1 or 2, and calculate the appropriate value in ACEify from it.

Reply:
Yep, thanks again for the catch.